pulp_ansible/app/tasks/collections.py

```diff
@@ -146,10 +169,16 @@ def import_collection(
     user_facing_logger = logging.getLogger("pulp_ansible.app.tasks.collection.import_collection")
 
     try:
-        with temp_file.file.open() as artifact_file:
+        with temp_file.file.open() as artifact_file, tarfile.open(
+            fileobj=artifact_file, mode="r"
+        ) as tar:
             importer_result = process_collection(
                 artifact_file, filename=filename, logger=user_facing_logger
             )
+            runtime = _get_file_obj(tar, "meta/runtime.yml", temp_file.file.name, False)
+            if runtime:
+                meta_runtime = yaml.safe_load(runtime)
+                importer_result["requires_ansible"] = meta_runtime.get("requires_ansible")
```

Comment:
@awcrosby does the importer handle `requires_ansible`?

Reply:
@fao89 currently it does not, I made issue https://issues.redhat.com/browse/AAH-231 to track this. We may end up making it a top-level key of the importer result as you have here.

Reply:
@fao89 to clarify, we will make the `galaxy-importer` provide this as a top-level key as you have in this PR, so this looks good

Reply:
thank you @awcrosby